docs/cloud/how-to-create-a-ticket-for-temporal-support.md

```diff
@@ -0,0 +1,31 @@
+---
+id: how-to-create-a-ticket-for-temporal-support
+title: How to create a ticket for Temporal Support
+sidebar_label: Support ticket
+description: To request assistance from Temporal Support, create a ticket in Zendesk.
+tags:
+  - explanation
+---
+
+:::info
+
+This procedure applies only to Temporal Cloud customers whose contracts include paid support.
+If you need assistance and don't have paid support, post your request in the [Temporal Community Forum](https://community.temporal.io) or the [Temporal Slack workspace](https://t.mp/slack).
```

Comment:
we have a specific #support-cloud channel that we might want to mention here
idk if we should put "paid support" here or not as iirc all customers have "paid support" by default. would check with sales team on this please

Reply:
Added name of channel. Checking with Sales.